src/Notifier.js

```diff
@@ -133,29 +153,35 @@ var Notifier = {
         }
 
         if(enable) {
-            if (!this.havePermission()) {
-                global.Notification.requestPermission(function() {
-                    if (callback) {
-                        callback();
+            // Case when we do not have the permission as 'granted'
+            if (this.isPermissionDefault()) {
```

Comment:
Well yeah normally it would never lead to this, as if `global.Notification.permission` != `default` the toolbar wouldn't even be shown.
But still in SDK shouldn't we have the case where we hide the toolbar in case the  `global.Notification.permission` is already `granted` or `denied`